static/resources/harness.js

```diff
@@ -323,7 +323,7 @@
     client.send(body);
     client.onreadystatechange = function() {
       if (client.readyState == 4) {
-        if (client.status >= 500) {
+        if (client.status >= 400) {
```

Comment:
@vinyldarkscratch what we should probably do is the equivalent of https://fetch.spec.whatwg.org/#ok-status, checking if that status is 2xx.

Reply:
Sounds good, opened #482 to do so!